_overviews/scala3-book/methods-most.md

```diff
@@ -241,37 +382,76 @@ def isTruthy(a: Any) =
     true
 ```
 
+{% endtab %}
+{% endtabs %}
+
 These examples show how that method works:
 
+{% tabs method_17 %}
+{% tab 'Scala 2 and 3' for=method_17 %}
+
 ```scala
 isTruthy(0)      // false
 isTruthy("")     // false
 isTruthy("hi")   // true
 isTruthy(1.0)    // true
 ```
 
-
+{% endtab %}
+{% endtabs %}
 
 ## Using `match` as a method body
 
 A `match` expression can also be used as the entire method body, and often is.
 Here’s another version of `isTruthy`, written with a `match` expression :
 
+{% tabs method_18 class=tabs-scala-version %}
+{% tab 'Scala 2' for=method_18 %}
+
+```scala
+def isTruthy(a: Any) = a match {
+  case 0 | "" | false => false
+  case _ => true
+}
+```
+
+{% endtab %}
+
+{% tab 'Scala 3' for=method_18 %}
+
 ```scala
 def isTruthy(a: Matchable) = a match
   case 0 | "" | false => false
   case _ => true
 ```
 
-This method works just like the previous method that used an `if`/`else` expression. We use `Matchable` instead of `Any` as the parameter's type to accept any value that supports pattern matching.
+> This method works just like the previous method that used an `if`/`else` expression. We use `Matchable` instead of `Any` as the parameter's type to accept any value that supports pattern matching.
 
-For more details on the `Matchable` trait, see the [Reference documentation][reference_matchable].
+> For more details on the `Matchable` trait, see the [Reference documentation][reference_matchable].
```

Comment:
I'm not sure why yet but it seems the code tabs plugin does not resolve these links - probably because it sees each tab as an individual markdown document, so I will make the link explicit
```suggestion
> For more details on the `Matchable` trait, see the [Reference documentation]({{ site.scala3ref }}/other-new-features/matchable.html).
```